pwa/src/templates/templateParts/resultsTemplates/layers/LayersResultTemplate.module.css

```diff
@@ -0,0 +1,41 @@
+.Container {
```

Comment:
All classNames may start with a lowercase letter.

Reply:
There's quite some hardcoded css values here. Since this component is not yet refined, I think having some hardcoded values is fine (as long as we are aware of them).

However, I do suggest creating some local variables, like so:

```
:root {
  --layer-component-padding: 14px;
  ... 
}
```

Reply:
Makes it easier for us to, when the time's there, extract this component into the component package.